<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditwindow</class>
 <widget class="QDialog" name="creditwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="window_name">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
     </property>
     <property name="text">
      <string>Окно кредитного калькулятора</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="window_name_8">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Сумма кредита&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="window_name_10">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберете тип кредита&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="month_or_years">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <item>
        <property name="text">
         <string>месяца</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>года</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="credit_sum">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="window_name_7">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Срок кредита&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="window_name_6">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Введите данные кредита&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="credit_type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Аннуитетный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дифференцированный</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="window_name_9">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Процентная ставка&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="rate">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="time">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="window_name_11">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Финальная &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;сумма кредита&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="text">
        <string>Последний месяц</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="final_sum">
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="window_name_14">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Результат расчета&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="final_rate_amout">
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="text">
        <string>Первый месяц</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="window_name_13">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Сумма переплаты&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="first_pay">
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="last_pay">
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QLabel" name="window_name_15">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Ежемесячный платаж&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="clear_data">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
       </property>
       <property name="text">
        <string>Очистить данные</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="result">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
       </property>
       <property name="text">
        <string>Расчитать кредит</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="back_tocalc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
       </property>
       <property name="text">
        <string>Вернуться в калькулятор</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
